lib/tsserver.js

```diff
@@ -44098,7 +44098,7 @@ var ts;
             }
         }
         function checkStrictModeNumericLiteral(node) {
-            if (inStrictMode && node.numericLiteralFlags & 32 /* Octal */) {
+            if (options.target < 1 /* ES5 */ && inStrictMode && node.numericLiteralFlags & 32 /* Octal */) {
```

Comment:
Updates to `lib/` should only happen as part of a separate LKG update; please revert this file

Reply:
@RyanCavanaugh Sorry about that, reverting now!